packages/core/test/smoke.test.ts

```diff
@@ -336,17 +336,19 @@ describe("shared can be accessed on both server and client", () => {
     FOO_BAR: "foo",
   };
 
-  const env = createEnv({
```

Comment:
Previously we called `createEnv` before "simulate" the `window` object below, which mean we already get `env` in server mode.

Nothing change to the test result, but at least the client-side simulation is fixed.